src/GraphQL/ParseGraphQLServer.js

```diff
@@ -55,6 +57,17 @@ class ParseGraphQLServer {
     app.use(this.config.graphQLPath, corsMiddleware());
     app.use(this.config.graphQLPath, bodyParser.json());
     app.use(this.config.graphQLPath, handleParseHeaders);
+    app.use(this.config.graphQLPath, (err, req, res, next) => {
+      if (err && err instanceof Parse.Error) {
+        res.json(
+          new ApolloError({
```

Comment:
Wouldn't it be better to use `this.parseGraphQLSchema.handleError()` function (https://github.com/parse-community/parse-server/blob/master/src/GraphQL/ParseGraphQLSchema.js#L102) or at least the same logic?

Reply:
I've refactored the conversion of the `Parse.Error` to `ApolloError`. Since the `handleError` method includes some logging and throws the error, I thought it was better to just refactor the middleware to use this conversion and set the error as response, because that's all we need for now.